microsoftgraph/microsoft-graph-explorer
# Specifies out-dir for the Typescript compiler

## Overview

Specifies out-dir for the  Typescript compiler

### Notes

Specifying an `out-dir` cleans up the work space and the developer does not have to configure their editor to ignore the source map files.
 
## Testing Instructions

* Do npm start
* After the browser is launched  observe that the out-dir folder has been created
